model/main.mod

```diff
@@ -25,11 +27,22 @@ set sUnpreferredCompanions within {sNurses, sNurses};
 set sPreferredSlots within {sNurses, sDays, sShifts};
 set sUnpreferredSlots within {sNurses, sDays, sShifts};
 
+# Binary indicator whether S consecutive shifts starting from given shift 
+# on given day are free for given nurse. 
+# This assumes that S shifts cover entire day, however length can be arbitrary.
+var vRest24hIndicator{sNurses, sWeeks, sWeekdays, sShifts}, binary;
+# Maximal allowed value of sum of vRest24hIndicator[n, w, *, *]:
+# there are S in each of 7 days; one oth them must be free
+param WORKED_24H_LIMIT := 6 * pNumberOfShifts;
```

Comment:
Does not follow `p<name>` naming convention + I think something along the lines of `pMaxDaysWithout24Break` would be more informative

Reply:
So I will rename it to `pMaxDaysWithout24BreakInWeek`.